javatests/arcs/core/data/DecodeManifestProtoTest.kt

```diff
@@ -10,15 +11,23 @@ import org.junit.runners.JUnit4
 @RunWith(JUnit4::class)
 class DecodeManifestProtoTest {
 
-    /**
-     * Validate that decoding of the proto encoded in JS works.
-     */
     @Test
-    fun decodesEncodedManifest() {
+    fun decodesManifestEncodedInTypeScript() {
         val path = runfilesDir() + "java/arcs/core/data/testdata/example.pb.bin"
         val recipe = RecipeEnvelopeProto.parseFrom(File(path).readBytes()).recipe
         assertThat(recipe.name).isEqualTo("PassThrough")
         assertThat(recipe.particlesList.map {it.specName}).containsExactly("Reader", "Writer")
         assertThat(recipe.handlesList.map {it.name}).containsExactly("thing")
     }
+
+    @Test
+    fun decodesManifestEncodedInTextFormat() {
+        val path = runfilesDir() + "java/arcs/core/data/testdata/example.pb.txt"
+        val builder = RecipeEnvelopeProto.newBuilder()
+        TextFormat.getParser().merge(File(path).readText(), builder)
+        val recipe = builder.build().recipe
+        assertThat(recipe.name).isEqualTo("PassThrough")
+        assertThat(recipe.particlesList.map {it.specName}).containsExactly("Reader", "Writer")
+        assertThat(recipe.handlesList.map {it.name}).containsExactly("thing")
```

Comment:
Nit: spaces inside curly braces: `{ it.specName }`. In the test case above too.

Reply:
Done :)